datastore/additional_data/sources/grant_metadata.py

```diff
@@ -0,0 +1,36 @@
+class GrantMetadataSource(object):
+    """Adds metadata to a grant:
+    * metadata/source_license
+    * metadata/source_license_name
+    """
+
+    ADDITIONAL_DATA_KEY = "metadata"
+
+    def __init__(self):
+        self._source_cache = {}
```

Comment:
now that the source_data isn't being queried for I'm not sure the cache doesn't anything?